### PR TITLE
Ht 21/update GitHub readme about your project

### PR DESCRIPTION
**Pull request by kalibria**
Update ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,57 @@
-# Getting Started with Create React App
+# Honda Tracker - booking app
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+The app for booking a car.
```
Comment (ikushlianski) [inline README.md]:
```suggestion
A demo app for booking a family car.
```

File: README.md [hunk 2]
```diff
@@ -1,46 +1,57 @@
...
-### `npm start`
+## General info
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+A simple app to track the time when users book Honda car. This application can be useful if several people use the same car.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+It is a responsive design app (mobile and desktop version).
```
Comment (ikushlianski) [inline README.md]:
```suggestion
It is a responsive app (mobile and desktop version).
```

File: README.md [hunk 3]
```diff
@@ -1,46 +1,57 @@
...
+After logged-in user see a calendar with booking information. User can create, edit, delete a booking, complete a ride, see a booking details, choose settings. All booking information comes from Mongo DB.
```
Comment (ikushlianski) [inline README.md]:
```suggestion
After logging in, the user sees a calendar with booking information. User can create, edit, delete a booking, complete a ride, see booking details, choose settings. All booking information comes from DynamoDB.
```

File: README.md [hunk 4]
```diff
@@ -1,46 +1,57 @@
...
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+* React
+* Typescript
+* CSS: Tailwind
+* Forms: Formik
+* Intl.DateTimeFormat
+* RTK query
+* Postman client for backend requests
```
Comment (ikushlianski) [inline README.md]: Would be nice to export a Postman collection and store the resulting json file in the app

**Review by ikushlianski — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kalibria) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (kalibria) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (kalibria) [reply README.md]: changed

File: README.md (hunk 4 above)
Comment (kalibria) [reply README.md]: changed

File: README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Honda Tracker - booking app
 
-The app for booking a car.
+A demo app for booking a family car.
```
Comment (ikushlianski) [inline README.md:3]:
```suggestion
A demo app for booking a family car.
```

```
test
```

Comment (ikushlianski) [inline README.md:3]:
```suggestion
TEST
```

**Review by ikushlianski — APPROVED** [review]

**Pull request merged**